pkg/compare/testdata/ReferenceDirectoryDoesntExist/localerr.golden

```diff
@@ -1,2 +1,2 @@
-error: "Reference directory doesn't exist"
```

Comment:
i think we can remove this test because now this one and "ReferenceConfigFileDoesntExist" are the same

Reply:
Done